apiserver/facades/client/bundle/mock_test.go

```diff
@@ -5,10 +5,48 @@ package bundle_test
 
 import (
 	"github.com/juju/testing"
+	"gopkg.in/juju/names.v2"
+
+	"github.com/juju/description"
+	"github.com/juju/juju/apiserver/facades/client/bundle"
+	"github.com/juju/juju/state"
 )
 
 type mockState struct {
 	testing.Stub
+	bundle.Backend
+}
+
+func (m *mockState) SetExportconfig(cfg state.ExportConfig) {
+	cfg.SkipActions = true
```

Comment:
This would be redundant, especially if this config never changes for bundle export.